pipeline-examples/output-pipelinerun.yaml

```diff
@@ -61,27 +61,27 @@ metadata:
   name: output-pipeline
 spec:
   resources:
-  - name: source-repo
-    type: git
+    - name: source-repo
+      type: git
   tasks:
-  - name: first-create-file          # 1. create file
-    taskRef:
-      name: create-file
-    resources:
-      inputs:
-      - name: workspace
-        resource: source-repo
-      outputs:
-      - name: workspace
-        resource: source-repo
-  - name: then-check                 # 2.  check file exists
-    taskRef:
-      name: check-stuff-file-exists
-    resources:
-      inputs:
-      - name: workspace
-        resource: source-repo
-        from: [first-create-file]
+    - name: first-create-file # 1. create file
```

Comment:
@sudhirverma It's mostly in the entire file. I don't think we need space fix here. Apart from extra spaces fix end of line. Please have a good look.